### DISCUSSION TITLE
Allow continuous calculation

### DISCUSSION THREAD

**Discussion by ItsCheeseGromit** [sadellie/unitto#139] [Ideas]
First off, great app that keeps getting better! Thank you.

My idea/request is to have a toggle option to allow continuous calculation. What I mean by this is, if I do a calculation, I'd like to be able to use the result in my next calculation instead of having to enter that result and then perform calculations against it.

In case that doesn't make sense, here's an example scenario:

Currently:
1 + 1 = 2
2 x 6 = 12

Continuous:
1 + 1 = 2 x 6 = 12

**Comment by sadellie** [comment]
https://github.com/sadellie/unitto/releases/tag/popstar
